draft-ietf-tls-subcerts.md

```diff
@@ -496,6 +500,7 @@ peer's expected identity.  It also takes the following steps:
 5. Use the public key in the peer's end-entity certificate to verify the
    signature of the credential using the algorithm indicated by
    DelegatedCredential.algorithm.
+6. Validate that DelegatedCredential.cred.valid_time is no more than 7 days.
```

Comment:
Should this be part of step (1), perhaps?

Reply:
Good idea!